compiler/cpp/src/generate/t_cpp_generator.cc

```diff
@@ -116,7 +116,8 @@ class t_cpp_generator : public t_oop_generator {
                                    bool read = true,
                                    bool write = true,
                                    bool swap = false,
-                                   bool stream = false);
+                                   bool stream = false,
+                                   bool is_struct = false);
```

Comment:
couple of solutions:
 - convert some of those bools into enum (especially that is_exception=true and is_struct=true can be set but it's a nonsense) [imho preferred]
- rename `bool stream` to `is_not_internal` or something like that (it was also added to distinguish internal structs and normal ones)
- do not bother that internal structs inherit from TBase, they're structs anyway